eclipse-collections-code-generator/src/main/resources/api/block/function/primitiveToPrimitiveFunction.stg

```diff
@@ -22,10 +22,19 @@ import java.io.Serializable;
  *
  * @since 3.0.
  */
+ <if(primitive1.specializedStream && primitive2.specializedStream && !sameTwoPrimitives)>@FunctionalInterface<endif>
 public interface <name1>To<name2>Function
-        extends Serializable
+        extends <if(primitive1.specializedStream && primitive2.specializedStream && !sameTwoPrimitives)>java.util.function.<name1>To<name2>Function, <endif>Serializable
 {
```

Comment:
I was expecting an else clause checking for sameToPrimitives and extending `<name1>UnaryOperator`. What do you think?